components/frontend/Dockerfile

```diff
@@ -1,18 +1,22 @@
+FROM node:23.1.0-alpine3.20 AS compile-image
+
+WORKDIR /home/frontend
+COPY public /home/frontend/public
+COPY package*.json /home/frontend
+COPY src /home/frontend/src
+COPY .env /home/frontend
+RUN npm install --ignore-scripts && \
+    npm run --ignore-scripts build
+
 FROM node:23.1.0-alpine3.20
 
 LABEL maintainer="Quality-time team <quality-time@ictu.nl>"
 LABEL description="Quality-time frontend"
 
 WORKDIR /home/frontend
-COPY package*.json /home/frontend
-COPY public /home/frontend/public
-COPY src /home/frontend/src
 COPY healthcheck.js /home/frontend
-COPY .env /home/frontend
-RUN npm install --ignore-scripts -g npm@10.9.0 && \
-    npm install --ignore-scripts && \
-    npm install --ignore-scripts -g serve@14.2.4 && \
```

Comment:
These two global installs are no longer needed? Nice!

Reply:
I removed the pinned install of npm because the base image has npm already. Serve is still needed, it's installed in the second stage.